operators/t8c/8.2.0/xls.charts.helm.k8s.io.crd.yaml

```diff
@@ -0,0 +1,3905 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Hi just would like to let you know that this version will not work for OCP 4.9 or k8s 1.22. 
See: https://github.com/turbonomic/t8c-install/issues/12

Reply:
hi @camilamacedo86 , yes I know